AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/AppOpenAdActivity.java

```diff
@@ -0,0 +1,112 @@
+package com.applovin.enterprise.apps.demoapp.ads.max;
+
+import android.os.Bundle;
+import android.view.View;
+
+import com.adjust.sdk.Adjust;
+import com.adjust.sdk.AdjustAdRevenue;
+import com.adjust.sdk.AdjustConfig;
+import com.applovin.enterprise.apps.demoapp.R;
+import com.applovin.enterprise.apps.demoapp.ui.BaseAdActivity;
+import com.applovin.mediation.MaxAd;
+import com.applovin.mediation.MaxAdListener;
+import com.applovin.mediation.MaxAdRevenueListener;
+import com.applovin.mediation.MaxError;
+import com.applovin.mediation.ads.MaxAppOpenAd;
+
+/**
+ * An {@link android.app.Activity} used to show AppLovin MAX App Open ads.
+ * <p>
+ * Created by avileung on 2023-02-10.
+ */
+public class AppOpenAdActivity
+        extends BaseAdActivity
+        implements MaxAdListener, MaxAdRevenueListener
+{
+    private MaxAppOpenAd appOpenAd;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState)
+    {
+        super.onCreate( savedInstanceState );
+        setContentView( R.layout.activity_app_open_ad_ );
```

Comment:
nit: Remove trailing `_` at the end of the file name (need to update xml file name too)